common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -589,6 +589,20 @@ def definition_to_xml(self, resource_fs):
 
         return xml
 
+    def create_youtube_url(self, youtube_id):
+        """
+
+        Args:
+            youtube_id: The ID of the video to create a link for
+
+        Returns:
+            A full youtube url to the video whose ID is passed in
+        """
+        if youtube_id:
+            return 'http://youtu.be/{0}'.format(youtube_id)
+        else:
+            return ''
```

Comment:
When you go to a `youtu.be` URL, it performs a redirect to the expanded `www.youtube.com` URL.  See for instance http://youtu.be/y8Kyi0WNg40 We should not force our users to make that extra request every single time.  It would be just as simple to send them directly to the destination URL: https://www.youtube.com/watch?v=y8Kyi0WNg40